### ISSUE TITLE
fix "scoop update -a"

### ISSUE THREAD

**Issue by AntonOks** [ScoopInstaller/Scoop#4530]
found [this one](https://github.com/ScoopInstaller/Scoop/commit/0d721abc2f200f9d0dda2fa26d7b68710a3a9840) finally merged (THANKS!!!) and was super excited to test it... 

PRO:
* ```scoop help update``` shows already the merged help info:
```powershell
λ  scoop help update
Usage: scoop update <app> [options]

'scoop update' updates Scoop to the latest version.
'scoop update <app>' installs a new version of that app, if there is one.

You can use '*' in place of <app> to update all apps.

Options:
  -f, --force               Force update even when there isn't a newer version
  -g, --global              Update a globally installed app
  -i, --independent         Don't install dependencies automatically
  -k, --no-cache            Don't use the download cache
  -s, --skip                Skip hash validation (use with caution!)
  -q, --quiet               Hide extraneous messages
  -a, --all                 Update all apps (alternative to '*')
  ```
* ```scoop update --all``` works great

CON: ```scoop update -a``` doesn't.. getting

```powershell
λ  scoop update -a
scoop update: Option -a requires an argument.
```

**Comment by rashil2000** [comment]
@filmor can you look into this?

**Comment by niheaven** [comment]
Fixed in #4531, will be closed while 'develop' being merged into 'master'.

**Comment by bjubes** [comment]
this has always been a bug, it wasn't introduced by the `-a` flag. see #4207. glad its finally being fixed.

**Comment by AntonOks** [comment]
btw., from what I see above, ```-a|--all``` is meant to be an alternative to '*' ... but it's not working the same way here...

If I do '*' all APPs are updated, if I do a '--all' only the scoop buckets are updated but __NOT__ the APPs?!?

**Comment by rashil2000** [comment]
Which branch are you using?

**Comment by niheaven** [comment]
Already fixed in #4531

**Comment by AntonOks** [comment]
> Which branch are you using?

what ever the default is :O (or in other words, didn't change it)
